common/src/main/resources/tachyon-default.properties

```diff
@@ -92,6 +92,10 @@ tachyon.worker.tieredstore.level.max=1
 tachyon.worker.tieredstore.level0.alias=MEM
 tachyon.worker.tieredstore.level0.dirs.quota=${tachyon.worker.memory.size}
 tachyon.worker.tieredstore.level0.dirs.path=/mnt/ramdisk
+tachyon.worker.tieredstore.level0.reserved.ratio=0.1
+tachyon.worker.space.reserver.enable=false
+tachyon.worker.space.reserver.period.ms=1000
```

Comment:
I guess the convention is to call `interval.ms` rather than `period.ms` in this property file.